src/db/seed/index.js

```diff
@@ -0,0 +1,49 @@
+import db from '..';
+import users from './userSeed';
+import motivators from './motivatorSeed';
+import getContents from './motivatorContentSeed';
+import getResults from './motivatorResultSeed';
+import getInputs from './motivatorInputSeed';
+
+const insertSeed = async (logger) => {
+  const execute = await db.models.user.findAll({
+    where: {
+      uid: users[0].uid,
+    },
+  });
+
+  if (execute) return;
```

Comment:
Maybe you could rename this variable to something more clear (e.g. `executed`, `ranInPast`, some actual good name you can come up with).